packages/governance/test/swingsetTests/contractGovernor/governedContract.js

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+
+import { Far } from '@agoric/marshal';
+import { E } from '@agoric/eventual-send';
+import { assert, details as X } from '@agoric/assert';
+import { sameStructure } from '@agoric/same-structure';
+
+import { buildParamManager, ParamType } from '../../../src/paramManager';
+
+const MALLEABLE_NUMBER = 'MalleableNumber';
+
+export const governedParameterTerms = {
+  contractParams: [MALLEABLE_NUMBER],
+};
+
+/* @type {ContractStartFn} */
+const start = async zcf => {
+  const { electionManager, governedParams } = zcf.getTerms();
```

Comment:
In reading this, I had to investigate to understand the types of these. If possible, could you type them?